scripts/kb_scripts/gwf_to_scs.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import os
+import sys
+from xml.etree.ElementTree import ParseError
+
+from termcolor import colored
+
+from support_scripts.gwf_parser import GWFParser
+from support_scripts.scs_writer import SCsWriter
+
+
+class Gwf2SCs:
+
+    GWF_FORMAT = '.gwf'
+    SCS_FORMAT = '.scs'
+
+    def __init__(self):
+        self.errors = []
+
+    @staticmethod
+    def collect_files(directory):
+        result = []
+
+        for root, _, files in os.walk(directory, topdown=False):
+            for f in files:
+                result.append(os.path.relpath(os.path.join(root, f), directory))
+
+        return result
+
+    def run(self, params):
+        print('Convert GWF to SCs...')
+        input = params.input
+        files = Gwf2SCs.collect_files(input)
+
+        file_id = 1
+        files = [file for file in files if os.path.splitext(file)[1] == Gwf2SCs.GWF_FORMAT]
+
+        for f in files:
```

Comment:
```suggestion
        for file_id, f in enumerate(files, start=1):
```
and you can remove file_id counter